zeta-cpi/package.json

```diff
@@ -1,16 +1,16 @@
 {
   "dependencies": {
-    "@project-serum/anchor": "^0.18.2",
+    "@project-serum/anchor": "^0.24.2",
     "@solana/spl-token": "^0.1.8",
-    "@zetamarkets/sdk": "^0.9.4",
-    "dotenv": "^10.0.0",
-    "node-fetch": "^3.1.0"
+    "@zetamarkets/sdk": "^0.16.8",
```

Comment:
0.16.14 is latest, shouldn't have any breaking changes so can just change this number